src/sinkhorn.rs

```diff
@@ -0,0 +1,30 @@
+use ndarray::{Array1, Array2, Axis};
+use float_ord::FloatOrd;
+
+pub fn sinkhorn(a: Array1< f32 >, b: Array1< f32 >, cost: Array2< f32 >, reg: f32) -> Array2< f32 >
+{
+    let n = cost.nrows();
+    let m = cost.ncols();
+    let epsilon = FloatOrd(1e-8);
+
+    let mut p = cost.mapv( |x| (-x / reg).exp());
+    p /= p.sum();
```

Comment:
Is necessary?

Reply:
Hmm, maybe not. It does rescale all the rows/cols iteratively, so I guess not

Reply:
Apparently not fba809d

Reply:
Possibly some numerical stability thing